app/controllers/katello/api/v2/flatpak_remotes_controller.rb

```diff
@@ -35,12 +34,14 @@ def index_relation
     end
 
     api :GET, "/flatpak_remotes/:id", N_("Show a flatpak remote")
+    api :GET, "/organizations/:organization_id/flatpak_remotes/:id", N_("Show a flatpak remote")
     param :id, :number, :desc => N_("Flatpak remote numeric identifier"), :required => true
+    param :organization_id, :number, :desc => N_("organization identifier")
     def show
       respond :resource => @flatpak_remote
     end
 
-    api :POST, "/flatpak_remote", N_("Create a flatpak remote")
+    api :POST, "/flatpak_remotes", N_("Create a flatpak remote")
     param :name, String, :desc => N_("name"), :required => true
     param :url, String, :desc => N_("url"), :required => true
```

Comment:
In reviewing the hammer side, I realized the "url" description is still a little confusing.  The RHEL 9 docs mention enabling the RHEL flatpak repo using the URL `https://flatpaks.redhat.io/rhel.flatpakrepo`, but I wonder if using the flatpakrepo file in our case would work. I've just been using `https://flatpaks.redhat.io/rhel`, but I'm not sure how a user would find that out.

I suppose these repos will be documented and we'll seed them into Katello at some point, but that might not work for other repos out in the wild.

Reply:
Ack..The flatpak index path is a little obscure unless it's a custom index users have on their end and know all the details. That's where the idea to auto-create the redhat and fedora came. If we do not auto-create them, we'll add some help text to list the 2 remote URLs or documentation.

Reply:
Would it be worth updating the URL description to be something like "flatpak index URL"? It looks like the code will accept a path to an index or will append the index suffix otherwise.

Reply:
Updated to: `Base URL of the flatpak registry index, ex: https://flatpaks.redhat.io/rhel/ , https://registry.fedoraproject.org/.`